packages/runtime-common/index-query-engine.ts

```diff
@@ -462,6 +464,21 @@ export class IndexQueryEngine {
     return { prerenderedCards, scopedCssUrls: [...scopedCssUrls], meta };
   }
 
+  async fetchCardTypeSummary(realmURL: URL): Promise<CardTypeSummary[]> {
+    let results = (await this.query([
+      `SELECT value
+       FROM realm_meta r
+       WHERE`,
+      ...every([
+        ['r.key = ', param(RealmMetaKey.CardTypeSummary)],
+        ['r.realm_url =', param(realmURL.href)],
+      ]),
+      `ORDER BY indexed_at`,
```

Comment:
presumably there is no need to order right? you should only be getting one value back (also you need to use realm_version as a predicate too).